activity-stream/send-to-device-rocket.html

```diff
@@ -0,0 +1,509 @@
+  }
+
+  #intro p {
+    margin: 0;
+  }
+
+  #intro h1 {
+    margin: 0 0 5px;
+    font-weight: bold;
+    color: var(--newtab-text-primary-color, #0f1126);
+  }
+
+  button.text-link {
+    border: none;
+    color: var(--newtab-button-primary-color, #0060df);
+    cursor: pointer;
+    display: inline;
+    padding: 0;
+    text-decoration: underline;
+  }
+
+  #email {
+    margin-right: 10px;
+    max-width: 400px;
+    padding: 8px;
+    width: 100%;
+  }
+
+  html[dir="rtl"] #email {
+    margin-left: 10px;
+    margin-right: 0;
+  }
+
+  @media (min-width: 544px) {
+    #email {
+      margin-bottom: 0;
+      width: 40%;
+    }
+  }
+
+  @media (min-width: 766px) {
+    #email {
+      width: 50%;
+    }
+  }
+
+  form button {
+    background: var(--newtab-button-primary-color, #0060df);
+    border: 0;
+    border-radius: 2px;
+    color: #f9f9fa;
+    cursor: pointer;
+    margin: 8px 0 0 0;
+    max-width: 400px;
+    padding: 8px 20px;
+    width: 100%;
+  }
+
+  @media (min-width: 544px) {
+    form button {
+      margin: 0 5px 0 0;
+      width: auto;
+    }
+  }
+
+  form input {
+    border: 1px solid var(--newtab-border-primary-color, #d7d7db);
+  }
+
+  form button:disabled {
+    cursor: default;
+    opacity: 0.5;
+  }
+
+  .disclaimer {
+    color: #777;
+    font-size: 12px;
+    margin: 10px 0 0;
+  }
+
+  @media (min-width: 766px) {
+    .disclaimer {
+      font-size: 14px;
+    }
+  }
+
+  #error {
+    color: #d70022;
+    display: block;
+    margin-top: 10px;
+  }
+
+  .alert {
+    background: #d70022;
+    border-radius: 50%;
+    color: #fff;
+    display: inline-block;
+    font-weight: bold;
+    height: 20px;
+    text-align: center;
+    width: 20px;
+  }
+
+  /* slide up stuff */
+  #snippets-container {
+    /* make sure entire snippet is hidden (1000px is excessive, but we don't know the height at this point) */
+    transform: translateY(var(--yOffset));
+  }
+
+  #snippets-container.visible {
+    transition: transform 0.25s ease;
+  }
+</style>
+
+<div class="snippet" id="snippet">
+  <div id="content-wrapper">
+    <section id="intro" class="section">
+      <div id="icon"></div>
+      <div class="section-content">
+        <section>
+        {% if title %}
+          <h1>{{ title }}</h1>
+        {% endif %}
+          <p>
+            {{ text|safe }}
+            <button type="button" class="text-link" id="show-form-button">{{ show_form_link_text }}</button>
+            or
+            <a href="{{ google_play_link_url }}" id="google-play-link">{{ google_play_link_text|default('visit the Play Store', true) }}</a>
+          </p>
+        </section>
+      </div><!--/.section-content-->
+    </section>
+
+    <section id="send-to" class="section">
+      <div class="section-content">
+        <form action="" id="send-to-device-form">
+          <input id="email" name="email" type="text" required="required" placeholder="{{ form_input_placeholder|default('Enter your email address...', true) }}" />
```

Comment:
From an a11y perspective, it would be nice to have a label associated with the email field here (even if hidden visually?)

Reply:
Good call. Done.